scripts/generate_latex_tables.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python3
+import sys
+import json
+
+def get_headers_for_baseline(domain):
+  return [
+    ("domain", "Domain"),
+    ("# rules found", "SlideRule Rules"),
+    ("rulefinding time (sec)", "Time (s)"),
+    ("# {} rules".format(domain), "Baseline Rules"),
```

Comment:
This is a bit hacky but I suppose it works, just need to make sure we change this and the json in sync